src/components/Rating.jsx

```diff
@@ -72,6 +72,13 @@ export default class Rating extends Component {
   }
 
```

Comment:
Fair enough. Feels somewhat weird to apply it directly to the DOM node. Might gut-feeling tells me this might cause trouble, but I don't have a real case. Let's go for it :+1: